### ISSUE TITLE
Some properties might contain HTML comments

### ISSUE THREAD

**Issue by magicDGS** [ygojson/ygojson-tools#62]
For example, we found the following property

```html
eu_release_date = March 2003<!-- \nhttps://web.archive.org/web/20060526080219/http:/www.esdeviumgames.com/newreleasesheets/Mar2nd2004web.pdf \nhttps://web.archive.org/web/20040325142347/http://www.theintoy.co.uk/product_info.php?cPath=1_27&products_id=167\n-->
```

on the [Structure Deck: Pegasus](https://yugipedia.com/wiki/Structure_Deck:_Pegasus) parse wikitext result (see it [here](https://yugipedia.com/api.php?action=parse&prop=wikitext&format=json&redirects=true&page=BIP&formatversion=2)).

This should be cleanup in case that we use fields as such and/or we find it on other places. We can either do it on `MarkupString` or some other mapping that contains the cleanup for those expected HTML format that are not useful for us.

Last but not least, we might consider some HTML-escaping library to perform this cleanup (maybe apache-commons have something that we might re-use, not only here).

**Comment by magicDGS** [comment]
Reduced priority to medium to evaluate if more escape/cleanup HTML is required and consider a library. Maybe also take into account if we should consider all String fields as `MarkupString` to be sure that HTML is not leaked into the output unless we really require it (i.e., the `ruby` japanese/korean syntax).

**Comment by magicDGS** [comment]
This should be implemented already as part of the wikitext-parsing algorithm implemented in #122. This should be tested anyway in case that it is not yet done. Keeping open to track.

**Comment by magicDGS** [comment]
Looks like the comments are not always cleanup by the parser, as the `single_letter_localization.approved.json` still contains the comment (probably because there are `\n` characters in the middle).
